### PR TITLE
#1304 icons for faith milestones

### LINKED ISSUE #1304: icons for faith milestones
On a contacts detail page, I would like to see icons representing each faith milestone.

The icon can be displayed in the milestone toggle buttons
image

Feature #1112 will also then use these icons to be displayed next to group member for those milestones that have been reached and are currently active in the group settings, e.g. 

image

### PR DESCRIPTION
**Pull request by squigglybob**
resolves #1304

### CODE REVIEW

**Comment by squigglybob** [comment]
@corsacca @ahillbilly currently looks like this.
image

File: dt-assets/scss/_details.scss [hunk 1]
```diff
@@ -29,6 +29,12 @@
         color:white;
     }
 }
+.milestones_select {
```
Comment (corsacca) [inline dt-assets/scss/_details.scss:32]: Please apply this to any multi_select that uses an icon

File: dt-contacts/dmm-module.php [hunk 2]
```diff
@@ -60,39 +60,48 @@ public function dt_custom_fields_settings( $fields, $post_type ){
                 "default" => [
                     "milestone_has_bible"     => [
                         "label" => __( 'Has Bible', 'disciple_tools' ),
-                        "description" => ''
+                        "description" => '',
+                        "image" => get_template_directory_uri() . "/dt-assets/images/bible.svg",
```
Comment (corsacca) [inline dt-contacts/dmm-module.php:64]: change 'image' to 'icon', to match other fields

**Comment by corsacca** [comment]
@squigglybob Thank you!
I like the icons :)

**Comment by squigglybob** [comment]
@corsacca :slightly_smiling_face: no worries, one thing to note is that as I don't have access to the nounproject account I have just copied and pasted in png's from searching icons on the site for now. Do they need to be downloaded officially as svg's?

**Comment by squigglybob** [comment]
@corsacca ok, I think those two notes on the code are resolved now

**Pull request merged**